lib/helpers.js

```diff
@@ -128,7 +128,10 @@ var fetchResource = function (resourceUrl, cb) {
     return cb(false);
   }
 
-  moduleForProtocol(resourceUrl).get(resourceUrl, function (response) {
+  moduleForProtocol(resourceUrl).get({
+    hostname: resourceUrl,
+    headers: { 'Origin': 'https://srihash.org/' },
```

Comment:
Yep, that would be great.
But I couldn't find a place in the helpers code that seems like a good point to get to `server` (as defined in index.js).